feature_mining/parse_and_model.py

```diff
@@ -12,20 +12,33 @@
 import time
 
 
-class ParseModel:
+class ParseAndModel:
     """
     Treats data input chain.
     Based on this data, computes matrices for reviews and features.
     """
 
-    def __init__(self):
+    def __init__(self, feature_list: list, filename: str, nlines: int = None, remove_stopwords: bool = False
+                 , start_line: int = 0, lemmatize_words: bool = True):
         """
         Constructor
         """
-        pass
+
+        # Run feature list formatter and save output
```

Comment:
Wait :)

Why are we doing this? Think I gotta check how effectively changing the nature of a function inside a class affects possible behavior. Guess it works but not so sure why this is done...